docs/platforms/unity/configuration/options/cli-options/index.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Sentry CLI Options
+description: "Learn more about how to configure the options used by Sentry CLI."
+sidebar_order: 30
+---
+
+## Programmatic Configuration
+
+Similar to the [programmatic configuration](/platforms/unity/configuration/options/programmatic-configuration/) of the SDK itself, the debug symbol upload via Sentry CLI also supports programmatic configuration.
+
+![Options Configuration](./img/sentry-cli-option-configuration.png)
+
+The `Configure` callback gets invoked during the build process and allows you to modify the options used by Sentry CLI.
```

Comment:
Is it worth clarifying a bit what is going on here? LIke we have a CLI, and it has options. But we also inlude the CLI in the SDK, and the options for it are configurable via the editor (right?) but also via C# code (what you already said below):

```suggestion
While the options below describe the arguments accepted by [Sentry CLI](https://docs.sentry.io/cli/), the Sentry SDK for Unity includes the CLI within it. And exposes a way to configure it through the editor itself, as well as through code. The `Configure` callback gets invoked during the build process and allows you to modify the options used by Sentry CLI.
```

Reply:
Updated!